### ISSUE TITLE
[CoE Starter Kit - QUESTION] Governance settings - Does 'Number Launches Last 30 Days' work independently 

### ISSUE THREAD

**Issue by MartinWGT** [microsoft/coe-starter-kit#7205]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

I'm setting up the governance settings and would like clarification for the item "Compliance - Apps - Number Launches Last 30 Day". Does this work independently of the other config settings? 
e.g. if the "Number Launches Last 30 Day" is set to 30, and an app was launched by the Maker 30 times while they are testing it, would that send the compliance request regardless of the number of shares?

![image](https://github.com/microsoft/coe-starter-kit/assets/142497418/e1680a9a-15c7-465c-8f21-6905de607bc1)

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

November 2023

### What app or flow are you having the issue with?

Wizard app

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Yes this an or situation. So if its launched > 30 times per month OR its shared with more than 6 people, then it will trigger the compliance flows.

**Comment by MartinWGT** [comment]
Thanks for confirming.
